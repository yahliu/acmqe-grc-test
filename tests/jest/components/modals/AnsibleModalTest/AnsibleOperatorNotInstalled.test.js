/* Copyright (c) 2020 Red Hat, Inc. */
/* Copyright Contributors to the Open Cluster Management project */

'use strict'

import { renderAnsibleOperatorNotInstalled } from '../../../../../src-web/components/modals/AnsibleAutomationModal/AnsibleOperatorNotInstalled'

describe('render AnsibleOperatorNotInstalled component', () => {
  it('AnsibleOperatorNotInstalled link renders as expected', () => {
    const component = renderAnsibleOperatorNotInstalled('us')
    expect(component).toMatchSnapshot()
  })
})
