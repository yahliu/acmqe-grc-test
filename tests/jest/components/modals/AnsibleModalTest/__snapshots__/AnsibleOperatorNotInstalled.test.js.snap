// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render AnsibleOperatorNotInstalled component AnsibleOperatorNotInstalled link renders as expected 1`] = `
<Alert
  actionClose=""
  isInline={true}
  title="The Ansible Automation Platform Resource Operator is required to create an Ansible job. Install the operator through the following link:"
  variant="danger"
>
  <AcmLaunchLink
    links={
      Array [
        Object {
          "id": "installAnsibleOperatorLink",
          "onClick": [Function],
          "text": "OperatorHub",
        },
      ]
    }
  />
</Alert>
`;
